.github/workflows/pr-comment-bot.yml

```
@@ -43,6 +43,7 @@ jobs:
 
             * 🙏 The Clarity team thanks you for opening a pull request
             * ⏳ The build for this PR has started
+            * 🍿 In the meantime, explore it in [StackBlitz](https://stackblitz.com/~/github.com/${{steps.get-pr-event.outputs.sourceHeadRepo}}/pull/${{steps.get-pr-event.outputs.pullRequestNumber}})
```

```suggestion
            * 🍿 In the meantime, explore it in [StackBlitz](https://stackblitz.com/~/github.com/${{github.repository}}/pull/${{steps.get-pr-event.outputs.pullRequestNumber}})
```

The PR `sourceHeadRepo` can point to a forked repository which will not work here.

---

correctred in commit 129e598